__tests__/api.test.js

```diff
@@ -65,4 +65,31 @@ describe('GET /api/reviews', () => {
             .get('/api/rewies')
             .expect(404)
     });
+});
+
+describe(' GET /api/reviews/:review_id', () => {
+    test('returns correct review', () => {
+        return request(app)
+            .get('/api/reviews/1')
+            .expect(200)
+            .then((res) => {
+                expect(res.body).toEqual({
+                    review_id: 1,
+                    title: 'Agricola',
+                    category: 'euro game',
+                    designer: 'Uwe Rosenberg',
+                    owner: 'mallionaire',
+                    review_body: 'Farmyard fun!',
+                    review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                    created_at: '2021-01-18T10:00:20.514Z',
+                    votes: 1
+                  })
+            })
+    });
+
+    test('404s on non existent id', () => {
+        return request(app)
+            .get('/api/reviews/1000')
+            .expect(404)
+    });
```

Comment:
Also worth putting in an error 400 test for an invalid data type being passes for review ID.